### ISSUE TITLE
Window movement lags when the window is partially off-screen

### ISSUE THREAD

**Issue by LoganDark** [stefansundin/altdrag#162]
I have a software-rendered window that I'm testing AltDrag with, and when it takes more than 1 frame to re-render, AltDrag has huge trouble moving the window back on-screen (from off-screen). It just kind of makes it difficult to move the cursor, applying some sort of friction or force to it to prevent me from moving the window back on-screen. It's extremely jarring and weird.

https://user-images.githubusercontent.com/4723091/185051705-c60d747d-2ca1-49c2-a851-b449c9b5a263.mp4

If I make the window smaller, or switch the blur to use the parallel/SIMD version to make it faster, then the issue goes away.

**Comment by spiralofhope** [comment]
Have you tried with "Mimic Aero Snap" disabled?

**Comment by LoganDark** [comment]
No, I stopped using this version for a fork that does the movement off-thread, this fixes the issue completely
